dataprofiler/tests/data_readers/test_csv_graph_data.py

```diff
@@ -207,17 +218,17 @@ def test_data_loader_edges(self):
             self.assertTrue(all_edges_present)
 
     def test_class_load(self):
-        for input_file in self.input_file_names_pos:
+        for input_file in self.file_or_buf_list:
             data = GraphData(input_file["path"])
-            self.assertEqual(type(data), GraphData)
+            self.assertIsNotNone(data.data)
```

Comment:
ensures data can be loaded as expected

Reply:
I don’t mind both but the new is better at validating